internal/lexer/token.go

```diff
@@ -0,0 +1,51 @@
+package lexer
+
+type TokenType int
+
+const (
+	EOF TokenType = iota
+	IDENTITY
+	NUMBER
+	STRING
+
+	PATH     // .
+	PIPE     // |
```

Comment:
Why pipe is not an operation?

Reply:
As for my understanding, we use the pipe to denote that from now on the source input is changed.
I see it as an operation.
So for example -
`routes.running.next-hop-interface:="br1"` - the source input is the current state.
`capture.base-iface-routes | routes.running.next-hop-interface:="br1"` - the source input capture.base-iface-routes.

Reply:
Is true that pipe can be considered an operator, maybe we can rename this flag since it's used here https://github.com/nmstate/nmpolicy/pull/2/files#diff-d1655fd776ca5fcf2821d7a478d18e80e68fc75d43a8aabeb5fbb07197c4f3ccR56

To denote that this is a binary operator and that It has to fill in the proper elements on the AST.